test/restart_test.go

```diff
@@ -58,18 +58,3 @@ func TestStopAfterCrashOfGammaServer(t *testing.T) {
 	require.NoError(t, err, "stop Gamma server should succeed")
 	require.True(t, strings.Contains(out, "Prism blockchain explorer stopped."), "Prism server should stop even if gamma crashed")
 }
-
-func TestStartedButNotReadyMessage(t *testing.T) {
```

Comment:
Why did you remove the test?

Reply:
This test is flaky because it runs gamma-server and hopes to call it before it runs. It seems too specific.